src/common/constants.js

```diff
@@ -1,22 +1,34 @@
-export const affiliate_signin_url = 'https://login.deriv.com/signin.php'
-export const affiliate_signup_url = 'https://login.deriv.com/signup.php'
+const isBrowser = () => typeof window !== 'undefined'
+
+const supported_domains = ['deriv.com', 'deriv.me']
+const domain_url =
+    isBrowser() && supported_domains.includes(window.location.hostname)
+    ? window.location.hostname
+    : 'deriv.com'
+
+export const deriv_app_url = `https://app.${domain_url}`
+export const deriv_bot_app_url = `https://app.${domain_url}/bot`
+export const deriv_dp2p_app_url = `https://app.${domain_url}/cashier/p2p`
+export const smarttrader_url = `https://smarttrader.${domain_url}`
+export const blog_url = `https://blog.${domain_url}`
```

Comment:
we are not supporting community and blog for deriv.me, please revert the change for those two links

Reply:
Hi @mustofa-binary , Resolved